### PR TITLE
add tag & issue date validation when aggregating

### PR DESCRIPTION
**Pull request by goranfr**
* validate tag / issue date when it's useful
* handle empty TS
* add tests for empty TS
* move fixtures to their own file
* add tests for TS aggregation

### CODE REVIEW

File: wapi/util.py [hunk 1]
```diff
@@ -138,29 +142,72 @@ def _rev_map_freq(frequency):
         return frequency
 
     @staticmethod
-    def sum(ts_list, name):
-        df = _ts_list_to_dataframe(ts_list)
-        return _generated_series_to_TS(df.sum(axis=1), name)
+    def sum(ts_list, name, data_type=TIME_SERIES):
```
Comment (haraldnh) [inline wapi/util.py]: I suspect `data_type` should rather be called `return_type` or similar.  And it should probably simply require that ts_list[0] has the necessary data and use that, and ignore whatever is set on the other elements.
